crates/optimism/node/src/txpool.rs

```diff
@@ -37,12 +37,22 @@ pub type OpTransactionPool<Client, S> = Pool<
 
 /// Pool transaction for OP.
 #[derive(Debug, Clone, derive_more::Deref)]
-pub struct OpPooledTransaction(EthPooledTransaction<OpTransactionSigned>);
+pub struct OpPooledTransaction {
+    inner: EthPooledTransaction<OpTransactionSigned>,
+    da_cost: U256,
```

Comment:
ah sorry I wasn't super clear in the issue but for https://github.com/paradigmxyz/reth/pull/13757 we don't need cost but gas usage estimation, which should happen lazily, so we need a LazyLock for this instead

`estimated_tx_compressed_size: LazyLock<u64>`